client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Schmoogle homepage</title>
+    <link rel="stylesheet" href="/client/stylesheet.css">
+    <script defer src="/client/DOM.js"></script>
+</head>
+<body>
+    <header>
+        <nav>
+          <ul id="nav_bar">
+            <li class="nav-links" id="gmail"><a href="#">Gmail</a></li>
+            <li class="nav-links"><a href="#">Images</a></li>
+            <li id="sign_in"><a href="#">Sign In</a></li>
+          </ul>  
+        </nav>  
+      </header>
+    <div class = 'bigtext'>
+        <section>
```

Comment:
I get what you were trying to do here. but it whould have been easyer have inported the image straight from the internet.  
e.g:
  <div class="#">
     <img src="https://www.google.co.uk/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png" alt="google logo">
  </div>
creat a div then simply put the image copied from google in there

Reply:
They didnt use the name google. Thats why they did their own letters